### DISCUSSION TITLE
Can't assembly UGS.zip with maven

### DISCUSSION THREAD

**Discussion by GabriellJohnny** [winder/Universal-G-Code-Sender#1944] [Help]
I'm trying to build the UGS plataform zip file with Maven but im geting an error everytime! here is the play by play of what is happening..

1- successfully build the project with  mvn install command on the comand prompt (Im using windows) 
2- tries mvn package assembly:assembly command and gets the following error
3 - Change Maven-assembly plugin and gets the same error

INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  03:05 min
[INFO] Finished at: 2022-07-24T06:17:51-03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-assembly-plugin:2.2-beta-5:assembly (default-cli) on project ugs-parent: Error reading assemblies: No assembly descriptors found. -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

C:\Users\Gabriell Jonas\Desktop\Universal-G-Code-Sender-2.0.11\Universal-G-Code-Sender-2.0.11>

Im using Windows 10 , Apache Maven 3.8.6 and  JDK 18
this is my Pom.Xml file , the only alteration done here was the bypass of the test routine of the surefire plugin.

[pom.zip](https://github.com/winder/Universal-G-Code-Sender/files/9175484/pom.zip)
![error](https://user-images.githubusercontent.com/88600585/180640996-3052226a-1d95-451d-a265-aa799169f86b.png)

**Comment by breiler** [comment]
What are you trying to build?

For the **UGS Classic** without tests you should use the following command (I have corrected the build instructions as they were wrong):
`mvn package -pl ugs-classic assembly:assembly -DskipTests=true`

For **UGS Platform** you should probably not use JDK 18 as it isn't compatible with the Netbeans platform version. I would recommend you to downgrade JDK 13, after that you can build a package with:
`mvn package -pl ugs-platform/application -P create-win64-package -DskipTests=true`
